src/main/java/pl/project13/maven/git/GitCommitIdMojo.java

```diff
@@ -66,6 +66,7 @@
   // these properties will be exposed to maven
   public static final String BRANCH = "branch";
   public static final String DIRTY = "dirty";
+  public static final String DIRTY_MARK = "dirty.mark";
```

Comment:
this breaks json output, I don't feel strongly enough about including this to break existing users.
If someone is using json output, they'd have:

```
dirty: false
```

And with your proposal they'd possibly break, since they could end up with:

```
dirty: { mark: ... }
```

so it's not a boolean anymore, and it's not possible to represent two values of `dirty` at the same time, thus we would end up breaking one or the other user.